.github/workflows/go-release.yml

```diff
@@ -70,6 +70,18 @@ jobs:
           git config --global user.name "GitHub Actions"
           git config --global user.email "actions@github.com"
 
+      - name: Get release directory name
+        id: release-dir
+        run: |
+          RELEASE_DIR_NAME=$(./scripts/go-release-automation.sh get_release_dir_name "${{ github.event.inputs.project-name }}" "${{ github.event.inputs.version }}")
+          echo "releaseDirName=$RELEASE_DIR_NAME" >> $GITHUB_OUTPUT
+
+      - name: Generate a changelog
+        uses: orhun/git-cliff-action@v4
+        with:
+          config: releases/go/.git-cliff.toml
+          args: --bump -u --prepend releases/go/${{ steps.release-dir.outputs.releaseDirName }}/CHANGELOG.md
```

Comment:
Note to reviewer: this command is not going to push. Its just going to prepend the changelog. The script does `git add *` in the release directory then commits and pushes.